docs/docusaurus/docusaurus.config.js

```diff
@@ -116,7 +116,7 @@ module.exports = {
             },
             {
               label: 'SLACK',
```

Comment:
we can't just update the link since the label is SLACK and it'll be incorrect.

we'll have to add another entry for DISCOURSE. you can ask @Rachel-Reverie for help.